### ISSUE TITLE
I can not import Image component from next/image

### ISSUE THREAD

**Issue by thang-phan** [kentcdodds/mdx-bundler#80]
- `mdx-bundler` version: ^5.2.1
- `node` version: v16.3.0
- `npm` version: 7.20.1

Relevant code or config

```js
import * as React from 'react'
import Image from 'next/image'

function EsstentialismGraph() {
  return (
    <div className="mt-4 flex justify-center">
      <Image src={'/essentialism_graph.svg'} 
        alt="Tư tưởng cốt lõi và không cốt lõi"
        layout="intrinsic" 
        width={250} 
        height={300} />
    </div>
    )
}

export default EsstentialismGraph
```

What you did:  Thank you for an amazing project! I try to import ``EsstentialismGraph`` component to the MDX file but it turns out I get an error about the process. I figured out the problem was I used `Image` component from next.

What happened:
<img width="968" alt="Screen Shot 2021-08-08 at 17 27 29" src="https://user-images.githubusercontent.com/85321214/128626012-a6b9c503-11f6-49f0-9eb1-30d3d246ee54.png">

Reproduction repository:

Problem description:

Suggested solution:

**Comment by chrislicodes** [comment]
- mdx-bundler version: ^5.2.1
- npm-version: 7.15.0
- node-version: v15.14.0

I also tried to bundle a component which is using

```js
 import { useRouter } from "next/router"
```

receiving following error-messages:

![image](https://user-images.githubusercontent.com/45950730/128639836-3633e321-336a-4e0a-bdb9-8640fe317d8a.png)

![image](https://user-images.githubusercontent.com/45950730/128639848-62dc1362-2c9d-43f7-9c16-b8585e534a91.png)

First I thought ChakraUI is the problem, but once I comment out the router it works fine.

**Comment by leerob** [comment]
In your code example, I don't see a `src` provided for the image. I'm able to use `next/image` with `mdx-bundler` as follows:

```jsx
<Image
  alt={`my image`}
  src={`/my-image.png`}
  width={2548}
  height={1640}
/>
```

**Comment by thang-phan** [comment]
Yes, It worked after  `Image` component is forwarded or is added as a global component.
@leerob I'm curious why I can't import `Image` in the MDX file directly as the way I did with `clsx` or `React`.

It works:
```tsx
import clsx from 'clsx'
import * as React from 'react'
```
But it doesn't work:
```tsx
import Image from 'next/image'
```

**Comment by nterol** [comment]
> As far as I am aware `next/image` needs to be [supplied as a global](https://github.com/kentcdodds/mdx-bundler#globals) so that it comes from the next.js app iinstead of being imported seperately. The same is true for `next/link`.

I'm not sure to understand how to do that. 
I added 
```
globals: {
      'next/image': 'nextImage',
      'next/link': 'nextLink',
    }
```
to the `bundleMDX` function and provided 
```
import NextImg from 'next/image';
import Link from 'next/link';
/* 
...
*/
getMDXComponent(code, {
        nextImage: NextImg,
        nextLink: Link,
      }) 
 ```
 to the page component but still receive this error whenever I import a component using next/image : 
 `ReferenceError: process is not defined`
 
 Am I missing something here ?

**Comment by jloesch30** [comment]
I was encountering a similar issue. It seems that adding the next component to the global list when defining `bundleMdx` was the trick such as below:
```javascript
      globals: {
        "next/router": "Router",
        "next/link": "Link",
        "next/image": "Image",
      },
```
And then when I call `getMdxComponent`, we can pass in the imported next components with their corresponding key to the value described above:
```javascript
  const Component = useMemo(
    () => getMDXComponent(code, { Link: Link, Image: Image, Router: Router }),
    [code]
  );
```

**Comment by vincerubinetti** [comment]
Ran into this with the `next/link` component. I had to do the `bundleMDX` `"globals"`, `getMDXComponent` `globals`, AND import the component in my MDX to get everything to work, though not sure if this is double-bundling the dependency.

If I don't import it in the mdx file, I get:

```
Error: Expected component `Link` to be defined: you likely forgot to import, pass, or provide it.
```

(Off topic) not pleased with the all hacks needed to get MDX working in Next.
